lib/SodiumCrypto.js

```diff
@@ -33,6 +33,11 @@ const BLOCK_SIZE = 65536
 const { Crypto } = ClientInterface
 
 export default class SodiumCrypto extends Crypto {
+  constructor() {
+    super()
+    this.mode = 'Sodium'
```

Comment:
I think I'd prefer this become a method that returns a string so that we can better enforce its required presence in the interface? More similar to a go interface.